package.json

```diff
@@ -18,6 +19,7 @@
     "lint": "jshint js preprocessing validation",
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
-    "validate": "node validation/markets-validator.js"
+    "validate": "node validation/markets-validator.js",
+    "start": "http-server -f index.html"
```

Comment:
No, Netlify takes care of that with the new `_redirects` file.

`http-server` is only needed locally so that we can have the same behaviour there.